src/easier68k/core/enum/op_size.py

```diff
@@ -0,0 +1,52 @@
+"""
+Op Size Enum
+Holds the values for various binary translations of size codes
+"""
+
+class BigSSize:
+    """
+    Represents the "Big S" sizes (dark purple on this chart: http://goldencrystal.free.fr/M68kOpcodes-v2.3.pdf)
+    """
+    BYTE = 0b01
+    WORD = 0b11
+    LONG = 0b10
+
+    @staticmethod
+    def parse(code: chr) -> int:
+        if code == 'B':
+            return BigSSize.BYTE
+        if code == 'W':
+            return BigSSize.WORD
+        if code == 'L':
+            return BigSSize.LONG
+
+class SmallSSize:
```

Comment:
I would be fine with naming this Size per the last comment

Reply:
Sounds good to me.